README.markdown

```diff
@@ -0,0 +1,4 @@
+# Nelson has moved!
+
+Please find Nelson at its new home: [getnelson/nelson](https://getnelson.github.io/nelson/)
```

Comment:
@adelbertc is this a bad merge? In this repo we don't want to change the repo from what it was IMHO

Reply:
Yes definitely a bad merge, will fix...